corehq/apps/userreports/views.py

```diff
@@ -739,6 +740,19 @@ def post(self, request, domain, data_source):
                 response_data = ConfigurableReportView.report_preview_data(self.domain, temp_report)
                 if response_data:
                     return json_response(response_data)
+                else:
```

Comment:
Does this happen for any error or specifically catch the filter error?

Reply:
Its specific to catch the filter error. @calellowitz